commit 243d8521b0164751e1ea8ed6322bf0c861298886
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-31T10:38:10-04:00 (10 weeks ago)
    
    TRITON-1849 portolan should use node v6
    TRITON-1850 update portolan base image to triton-origin-x86_64-18.4.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 6cff8eb..6f9426f 100644
--- a/Makefile
+++ b/Makefile
@@ -44,9 +44,9 @@ CTF_TYPES=-t svp_req_t \
 TAPE=node_modules/.bin/tape
 
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_VERSION=v0.10.48
-	NODE_PREBUILT_TAG=zone
-	NODE_PREBUILT_IMAGE=de411e86-548d-11e4-a4b7-3bb60478632a
+	NODE_PREBUILT_VERSION=v6.17.0
+	NODE_PREBUILT_TAG=zone64
+	NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 ENGBLD_USE_BUILDIMAGE	= true
@@ -71,7 +71,8 @@ COMMIT=$(shell git describe --all --long  | awk -F'-g' '{print $$NF}')
 RELEASE_TARBALL:=$(NAME)-pkg-$(STAMP).tar.gz
 RELSTAGEDIR:=/tmp/$(NAME)-$(STAMP)
 
-BASE_IMAGE_UUID = de411e86-548d-11e4-a4b7-3bb60478632a
+# our base image is triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SDC Portolan Service
 AGENTS		= amon config registrar
diff --git a/package.json b/package.json
index d9f8136..28fdf1e 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "portolan",
     "description": "SDC Portolan service",
-    "version": "0.1.5",
+    "version": "0.2.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
@@ -10,7 +10,7 @@
         "clone": "1.0.0",
         "cmdln": "3.0.1",
         "ctype": "0.5.3",
-        "dtrace-provider": "0.3.1",
+        "dtrace-provider": "0.8.7",
         "ipaddr.js": "0.1.6",
         "lomstream": "^1.1.0",
         "macaddr": "0.0.1",
